### DISCUSSION TITLE
When hovering over osu! logo in title screen, the first panel should extend

### DISCUSSION THREAD

**Discussion by Phippe** [ppy/osu#13035] [Ideas]
**Describe the bug:** I am used to just click on the logo to get into the game. Now when I want to play multiplayer or have a look at the editor, I aim more or less at the area between the red bars seen on the first screenshot, as it is in the middle. But as soon as I move the cursor and hover over the first panel, it extends to right. What used to be the middle now is the left of the second or even the right of the first panel, making it quite possible to misclick.

My proposition is that the first panel extends as soon as you hover over the logo, so you'll aim at the correct position the first time.

Hope this is the right category for this kind of thing, couldn't decide between this and feature request.

**Screenshots or videos showing encountered issue:** 
![grafik](https://user-images.githubusercontent.com/34287258/60678757-d43b3600-9e85-11e9-966a-d17f1c830833.png)
![grafik](https://user-images.githubusercontent.com/34287258/60678767-d8ffea00-9e85-11e9-89fa-3690bb01d118.png)

**osu!lazer version:** 2019.704.0

**Comment by peppy** [comment]
Would need to implement this and see how it feels.

**Comment by Phippe** [comment]
The reason for just clicking on the logo was that on stable, it wasn't possible to just click on the same spot of the play button to enter the game. After playing a little more, I realized that this is now possible, making it not really necessary to click on the logo. So if it's too cumbersome to implement just to test it, feel free to close this issue.
